src/context/WalletProvider/NativeWallet/components/LegacyLogin.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Button,
+  Checkbox,
+  FormControl,
+  FormErrorMessage,
+  Input,
+  ModalBody,
+  ModalHeader,
+  useColorModeValue,
+} from '@chakra-ui/react'
+import { useState } from 'react'
+import { FieldValues, useForm } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { RouteComponentProps } from 'react-router-dom'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text'
+
+// @TODO(NeOMakinG): Remove this and add the 2fa code logic
+const twoFactorAuthCode = ''
+
+export const LegacyLogin = ({ history }: RouteComponentProps) => {
+  const [isCaptchaValidated, setCaptchaValidated] = useState(false)
+  const captchaBgColor = useColorModeValue('gray.50', 'gray.700')
+  const checkboxBorderColor = useColorModeValue('gray.400', 'white')
+
+  const {
+    handleSubmit,
+    register,
+    reset,
+    formState: { errors, isSubmitting },
+  } = useForm({ shouldUnregister: true })
+
+  const translate = useTranslate()
+
+  const onSubmit = async (values: FieldValues) => {
+    // Reset every fields in order to remove the password and email from the memory
+    reset()
+    return
+  }
+
+  const onCaptcha = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
This doesn't indicate much: `onCaptcha` what?
From the spec and this PR, it looks like the intent is `onCaptchaRequested` ?